lottie/src/main/java/com/airbnb/lottie/network/LottieNetworkCacheProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.airbnb.lottie.network;
+
+import java.io.File;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Interface for providing the custom cache directory where animations downloaded via url are saved
+ */
+public interface LottieNetworkCacheProvider {
+
+  /**
+   * Called during cache operations
+   *
+   * @return cache directory
+   */
+  @NonNull File getCacheDir();
```

Comment:
You can make this `@Nullable` and pass in the url to enable apps to tell Lottie to skip the cache for a particular animation.

Reply:
Hey!
LottieNetworkCacheProvider is responsible for the concrete implementation of CacheProvider, it cannot have @Nullable File getCacheDir(). It's weird to have a CacheProvider that doesn't provide cache. I looked at the library code and see that cacheKey
 == null is used to skip the cache for an animation. We can continue to use this logic.

Reply:
SGTM